5-substitutions.sh

```diff
@@ -0,0 +1,35 @@
+mkdir subs
+cd subs
+
+# q1
+touch File{A,B,C,D,E,F}.txt
```

Comment:
This works, but you were expected to use the ".." subtitution like this: `touch File{A..F}.txt`